services/network/setup.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+
+import os
+import ssl
+from setuptools import setup
+
+# Ignore ssl if it fails
```

Comment:
when does it fail? Should we be concerned?

Reply:
I got this error while running setup.py in our companion image, probably an issue related to the old ssl libraries in python3 maybe.